package.json

```diff
@@ -28,6 +28,7 @@
     "storybook": "storybook dev -p 6006",
     "storybook:build": "storybook build -o out/storybook-static",
     "build:preview":"SSG=false next build",
+    "next:start":"yarn start",
```

Comment:
you added `build:preview` and `start` to this file prior. those are the two new commands needed, so this is redundant (and would simply call the `start` command one line lower)